security-release-process-documentation/security-release-process.md

```diff
@@ -258,3 +297,346 @@ Yes.
 CrashOverride will vouch for Acidburn joining the list on behalf of the "Seven"
 distribution.
 ```
+
+## Security Bug Bar
+
+The content presented below outline basic criteria when considering the effect
+of bugs disclosed to the Product Security Team and their severity. These apply
+mostly to items in "core" but could be abstractly applied to those in other
+repos as well.
+
+### Server
+
+The server bar is usually not appropriate when user interaction is part of 
+the exploitation process. If a "Critical" vulnerability exists only on server 
+components, and is exploited in a way that requires user interaction and results 
+in the compromise of the server, the severity may be reduced from "Critical" to 
+"Important" in accordance with the data definition of extensive user 
+interaction presented at the start of the client severity pivot.
+
+#### Critical
+
+Network worms or _unavoidable_ cases where the server is "owned."
+
+**Elevation of privilege** 
```

Comment:
Nit: these subtypes of "Critical" end up rendering similarly to "Critical" Maybe do the 5-hash thing?

CURRENT:
#### Critical

Network worms or _unavoidable_ cases where the server is "owned."

**Elevation of privilege**

PROPOSED:
#### Critical

Network worms or _unavoidable_ cases where the server is "owned."

##### Elevation of privilege

Reply:
omg i was sitting here like effing a markdown is the worst hahaha, fixing